### PR TITLE
"More" screen

### PR DESCRIPTION
**Pull request by madalinazanficu**
Designed "More" screen as per figma.

### CODE REVIEW

File: mobile/app/(app)/(drawer)/(tabs)/more.tsx [hunk 1]
```diff
@@ -1,10 +1,105 @@
...
+
+interface SectionProps {
+  headerText: string;
+  subHeaderText?: string;
+  iconName: string;
+  chevronRight?: boolean;
+  action?: () => void;
+}
+
+const Section = (props: SectionProps) => {
```
Comment (birloiflorian) [inline mobile/app/(app)/(drawer)/(tabs)/more.tsx:77]: I would like this to be called MenuItem, update the props interface also

File: mobile/app/(app)/(drawer)/(tabs)/more.tsx [hunk 2]
```diff
@@ -1,10 +1,105 @@
...
+    </Screen>
+  );
+};
+
+interface SectionProps {
+  headerText: string;
+  subHeaderText?: string;
+  iconName: string;
+  chevronRight?: boolean;
+  action?: () => void;
```
Comment (birloiflorian) [inline mobile/app/(app)/(drawer)/(tabs)/more.tsx:74]: Update to: {label: string; helper?: string; icon: string; chevronRight?: boolean; onClick: () => void; }

File: mobile/app/(app)/(drawer)/(tabs)/more.tsx [hunk 3]
```diff
@@ -1,10 +1,105 @@
...
+
+  const [isPressed, setIsPressed] = useState(false);
+
   return (
-    <View>
-      <Text>More</Text>
-    </View>
+    <Card
+      onPress={action}
+      onPressIn={() => setIsPressed(true)}
```
Comment (birloiflorian) [inline mobile/app/(app)/(drawer)/(tabs)/more.tsx:87]: Out of curiosity can we extend the card for this case using the styled method the same as the button to have the pressedStyle?

File: mobile/app/(app)/(drawer)/(tabs)/more.tsx
Comment (birloiflorian) [inline mobile/app/(app)/(drawer)/(tabs)/more.tsx:1]: Looks good! Good job.

**Review by birloiflorian — CHANGES_REQUESTED** [review]

File: mobile/app/(app)/(drawer)/(tabs)/more.tsx (hunk 1 above)
Comment (madalinazanficu) [reply mobile/app/(app)/(drawer)/(tabs)/more.tsx]: done

File: mobile/app/(app)/(drawer)/(tabs)/more.tsx (hunk 2 above)
Comment (madalinazanficu) [reply mobile/app/(app)/(drawer)/(tabs)/more.tsx]: Done

File: mobile/app/(app)/(drawer)/(tabs)/more.tsx (hunk 3 above)
Comment (madalinazanficu) [reply mobile/app/(app)/(drawer)/(tabs)/more.tsx]: Yes, looks good.

**Review by birloiflorian — APPROVED** [review]

**Pull request closed without merging**